src/bin/mosaic-start.ts

```diff
@@ -132,6 +133,15 @@ mosaic
 
         await (new Graph(graphDescription).start());
       }
+      // printing of endpoints on console.
+      const endPointsTable = markdownTable([
+        ['', 'rpc port', 'ws port'],
+        [chain, rpcPort, websocketPort],
+        ['Graph node', graphDescription.rpcPort, graphDescription.websocketPort],
```

Comment:
this line would be unnecessary and even confusing to some people using `--withoutGraphNode` flag. Print it only when this flag isn't used.